docs/spec/abci/abci.md

```diff
@@ -296,6 +296,12 @@ Commit are included in the header of the next block.
 
 - **Request**:
   - `Tx ([]byte)`: The request transaction bytes
+  - `Type (CheckTxType)`: What type of `CheckTx` request is this? At present,
+    there are two possible values: `CheckTx_Unchecked` (the default, which says
```

Comment:
Oh, seems this is not aligned with the changes actually made to the proto file, ie. the types are `New` and `Recheck`

Reply:
We should actually also add an entry to `docs/spec/abci/apps.md` explaining the impact of this field on application developers (ie. they may be able to skip signature verification)